### DISCUSSION TITLE
When updating Add-ons in EE6, remain on Update pane if other upgrades still available

### DISCUSSION THREAD

**Discussion by jcogs-design** [ExpressionEngine/ExpressionEngine#2860] [Feature Requests]
When updating add-ons in EE6 you have to open a specific pane ("Update").
Following completion of an add-on update processing is returned to the main Add-on Panel, even if there are other add-ons requiring updates.
Workflow would be improved if when there are multiple add-ons to update if processing returned to the "update" pane after an add-on is updated.

**Comment by litzinger** [comment]
I think the new add-ons page, while it had good intentions, is a step backwards. I can no longer batch update multiple add-ons, and as jcogs mentions, have to repeatedly click the Updates tab to see what needs to be updated. I'd prefer to see a list of add-ons, similar to what was in EE 5, but with the icons. Add-ons needing to be updated, sorted to the top so they're immediately visible, and add back the option to update multiple add-ons at once.

**Comment by intoeetive** [comment]
Getting this implemented in https://github.com/ExpressionEngine/ExpressionEngine/pull/4078

Also not too much related, rather just wondering - it if would be worth moving uninstalled add-ons into a separate tab
